### PR TITLE
Set the RNG seed for each run through scall/pcall

### PR DESCRIPTION
**Pull request by SwadicalRag**
valve pls fix

### CODE REVIEW

**Comment by willox** [comment]
why

**Comment by SwadicalRag** [comment]
TL;DR: the user experience from functions which rely on math.random() is severely impeded without it.

For example, upon a `.restart`, `cookie.maymays.lenny()` would always return `( ͡☉ ͜ʖ ͡☉)`, and the specific lenny face would be overused, and the meme-quality™ of the lenny function would deteriorate over time.

The `cock` library is also dependent upon `math.random()` to initialise cock data. If the seed for math.random wasn't set, everyone would likely have `cock`s of the same base size.

I realise that you can call `math.randomseed` from said functions, but setting the seed should be a standard operation, in my opinion.

**Comment by willox** [comment]
I thought you were calling math.randomseed every time Lau ran, this makes actual sense though.
